siliconcompiler/utils.py

```diff
@@ -302,3 +302,13 @@ def format_fileset_type_table(indent=12):
         table += f"{indent}{filetype:<10}| {fileset:<11}| {ext}\n"
 
     return table
+
```

Comment:
It's ok, but this one feels awkward. I agree that we should try to not repeat ourselves, but this is coupling/hardcoding settings for specific tools inside the utils flow. How about putting this util function inside the target dir?

Reply:
sure, I can move it there.